stdlib/Distributed/src/cluster.jl

```diff
@@ -959,16 +959,6 @@ function _rmprocs(pids, waitfor)
 end
 
 
-struct ProcessExitedException <: Exception end
-
-"""
-    ProcessExitedException()
-
-After a client Julia process has exited, further attempts to reference the dead child will
-throw this exception.
-"""
```

Comment:
It would be good to add this docstring to the new definition in Base (updated accordingly) and make sure all of the docs reference the right thing.

Reply:
The language used here should just be copied verbatim into the new docstring.